common/presenters/moves-by-vehicle.js

```diff
@@ -0,0 +1,77 @@
+const { addMinutes, formatISO, isToday, parseISO } = require('date-fns')
+const { isEmpty, get, groupBy, orderBy, sortBy } = require('lodash')
+
+const setMissingToLocation = require('../helpers/move/set-missing-to-location')
+
+const moveToCardComponent = require('./move-to-card-component')
+
+module.exports = function movesByVehicle(data, locationKey = 'to_location') {
+  const locations = []
+  const groupedByLocation = groupBy(data, 'meta.vehicle_registration')
+
+  Object.entries(groupedByLocation).forEach(([group, moves], index) => {
+    const vehicleRegistration = get(moves[0], 'meta.vehicle_registration')
```

Comment:
I assume every accepted move has a vehicle registration?

Reply:
Does `get` support the `[0]` property as the start of the path?

Reply:
I think it could be used in either instance in this case. But what this is doing is looking for the property path `meta.vehicle_registration` within the first move.

This is based on the assumption that because these moves have already been grouped by this property, that they all contain the same registration. So within this block I'm assigning the value from the first move to this variable.

Reply:
>I assume every accepted move has a vehicle registration?

@Nimphal not necessarily. However, that's where the fallback comes in. So this line is just taking whatever the value is and assigning it to a variable.

However, thinking about it, I could use `group` which should provide the same value.